packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.test.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import { ShimmeredDetailsList } from './ShimmeredDetailsList';
+
+// Populate mock items for testing
+function mockItems(count: number): any {
+  const items = [];
+
+  for (let i = 0; i < count; i++) {
+    items.push({
+      key: i,
+      name: 'Item ' + i,
+      value: i
+    });
+  }
+
+  return items;
+}
+
+describe('ShimmeredDetailsList', () => {
+  it('renders List correctly', () => {
+    ShimmeredDetailsList.prototype.componentDidMount = jest.fn();
+
+    const component = renderer.create(
+      <ShimmeredDetailsList
+        items={mockItems(5)}
+        // tslint:disable-next-line:jsx-no-lambda
+        onRenderRow={() => null}
```

Comment:
Why not just use jest.fn() in places like this?

Reply:
I just cloned this from DL's test... didn't seem to be a big deal to do a jest.fn() or have props be passed in, in this case

Reply:
Agree it's not a big deal - but you can avoid the tslint disable this way :) Again, no big deal.